src/amo/components/LandingPage.js

```diff
@@ -108,9 +112,43 @@ export class LandingPageBase extends React.Component {
     }
 
     const { addonType, html } = content;
+    const themeText = i18n.gettext(
+      oneLine`Change your browser's appearance. Choose from thousands of themes to give Firefox the
+      look you want.`);
+    const extensionsText = i18n.gettext(
+      oneLine`Install powerful tools that make browsing faster and safer, add-ons make your browser
+      yours.`);
 
     return (
       <div className={classNames('LandingPage', `LandingPage-${addonType}`)}>
+
+        <div className="LandingPage-header">
+          <div className="LandingPage-header-top">
+            <Icon name={visibleAddonType} />
+            <div className="LandingPage-header-text">
+              <h1 className="LandingPage-heading">
+                {addonType === ADDON_TYPE_THEME
```

Comment:
This means anything not a theme gets output as an extension but that's not 100% true. If this were a dictionary or open search plugin it would say extension. I guess that's kinda how we treat them, but I'd rather see a mapping for this similar to what we do elsewhere so it's "Theme"/"Extension"/"Add-on" if unknown.